spec/miscellaneous/lib/html_cleaner_spec.rb

```diff
@@ -198,6 +210,11 @@
           result = sanitize_value(field, value)
           expect(result).to eq("")
         end
+        it "should strip video" do
```

Comment:
Could you add a blank line above this and just make it `it "strips video" do` without the `should`?